backend/api/views.py

```diff
@@ -103,4 +106,23 @@ def uptake(request):
     """
     Take a JSON payload from telemetry streaming to record experiment uptake.
     """
+    payload = request.data.get('uptake')
```

Comment:
If the uptake key is missing this will raise an exception, should we either try/catch or use .get('uptake', None) and check for None?

Reply:
If it's not there `.get()` returns `None` by default. And then in the check below, the `if not payload` will raise a `ValidationError`.

Reply:
So you are right!  I seem to remember it raising an error, but I'm probably confusing it with model.objects.get